apps/kampus/app/pano/features/post-list/MoreOptions.tsx

```diff
@@ -118,6 +118,7 @@ export const MoreOptionsDropdown = (props: Props) => {
           )}
           <DropdownMenuItem
             onSelect={() => {
+              navigator.clipboard.writeText(`pano/post/${post?.id}`);
```

Comment:
we should remove pano part because the url should like this: pano.kamp.us/post/${post.id}